modules/data-governance/versions.tf

```
@@ -16,4 +16,22 @@
 
 terraform {
   required_version = ">= 0.13"
+  required_providers {
+
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 3.77"
+    }
+
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = "~> 3.77"
+    }
+
+    random = {
+      source  = "hashicorp/random"
+      version = "3.4.3"
```

Avoid breaking change

```suggestion
      version = ">= 2.3"
```